invenio_rdm_records/resources/serializers/csl/schema.py

```diff
@@ -77,6 +77,20 @@ def get_type(self, obj):
         props = resource_type_record["props"]
         return props.get("csl", "article")  # article is CSL "Other"
 
+    def get_doi(self, obj):
```

Comment:
I think this implementation is identical to https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/serializers/schemas.py#L16

Therefore, it can be removed and it would suffice adding the Mixin to the inheritance in line 48:

```python
class CSLJSONSchema(Schema, CommonFieldsMixin):
```

Example usage [BibTexSchema](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/serializers/bibtex/schema.py#L29).